benchmarks/nemo/nemolite2d/kernels/c_family/continuity_kern.c

```diff
@@ -123,13 +124,10 @@ void continuity_code(int ji, int jj,
 #endif
     /* Locals */
     double rtmp1, rtmp2, rtmp3, rtmp4;
-    int idxim1, idxjm1;
-    int idx = jj*width + ji;
-
-    if(jj == 0)return;
 
```

Comment:
I removed it because I didn't know what it does and there isn't any equivalent expression in the Fortran kernel. However, I see there are equivalent expressions in some of the OpenCL pre-processor blocks in other kernels, so I re-introduced it just inside the OpenCL for consistency.
(At some point I will try to use again the 'single_file_c_opencl' version that uses this kernels and see if everything still works and if this expressions can be removed)